flowTypes.js

```diff
@@ -0,0 +1,50 @@
+// @flow
+
+declare module 'react-native-google-signin' {
+  declare type ConfigureParams = {
+    iosClientId?: string,
+    offlineAccess?: boolean,
+    webClientId?: string,
+    scopes?: string[],
+    hostedDomain?: string,
+    forceConsentPrompt?: boolean,
+    accountName?: string,
+  };
+
+  declare type HasPlayServicesParams = {|
```

Comment:
I wonder if it makes sense to use the exact syntax (`{| ... |}`) for other declared types as well.. wdyt?